index.html

```diff
@@ -10,11 +10,19 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
     <link rel="stylesheet" href="css/style.css" />
     <title>StopWatch</title>
   </head>
   <body>
+    <div class="container">
```

Comment:
@Farhan-Ali2002 please remove this container, as u can directly work on the body and u have it already selected in at the top of the .js file(DOM tree).